client/components/activity-card-list/visible-days-limit-upsell/hooks.ts

```diff
@@ -41,7 +41,7 @@ export const useTrackUpsellView = (
 			}
 
 			dispatch(
-				recordTracksEvent( 'calypso_activitylog_retentionlimit_view', {
+				recordTracksEvent( 'calypso_activitylog_visiblelimit_upsell_view', {
```

Comment:
It's possible that changing these Tracks events will break a funnel or report somewhere, so I wonder if we should leave the Tracks events as they are. I'm not sure if there is a way to look for funnels that use existing Tracks events.

Reply:
These are brand new Tracks events I made in a recent PR, so none of these have been used in production yet. 🙂